test/error.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Nuts node
+ * Copyright (C) 2021. Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package test
+
+import (
+	"errors"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// AssertIsError asserts that expected is, or is the cause of the given actual error (according to errors.Is()).
+func AssertIsError(t *testing.T, actual error, expected error) bool {
```

Comment:
How does this differ from : https://github.com/stretchr/testify/blob/dc5c261377d971ad796211563f0e3e73954af5e6/assert/assertions.go#L1734

Reply:
Not very much. I upgraded the lib and now we can use it 👍  (finally it's there)